app/views/restaurants/show.html.erb

```diff
@@ -0,0 +1,19 @@
+
+<div class="container">
+
+  <%= render 'restaurants/header' %>
+
+  <div class="row">
+
+      <div class="col-md-8">
+          <img class="img-responsive" src="<%= @restaurant.image_url %>" alt="<%= @restaurant.name %>" style="width:700px;height:400px;">
```

Comment:
use this

http://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag